src/common/actions/register-name.js

```diff
@@ -75,21 +85,24 @@ async function signAgreement(root, discharge) {
   }
 }
 
-export async function internalRegisterName(root, discharge, snapName,
-                                           repositoryUrl) {
-  const response = await fetch(`${BASE_URL}/api/store/register-name`, {
+async function requestRegisterName(root, discharge, snapName) {
+  return await await fetch(`${BASE_URL}/api/store/register-name`, {
```

Comment:
Oh... that's interesting that it doesn't throw syntax error and actually works...